### PR TITLE
Add ng-repeat in product list and productCtrl

### PR DESCRIPTION
**Pull request by PatrykDawczak**

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ angular
       $stateProvider
           .state('view1', {
             url: '/',
-            templateUrl: 'view1/view1.html'
+            templateUrl: 'view1/view1.html',
+            controller: 'productCtrl'
```
Comment (TomaszGabrysiak) [inline app/app.js]: Pls rename it into ProductCtrl. It's good practice

File: app/view1/view1.js [hunk 2]
```diff
@@ -1,14 +1,5 @@
...
-}])
-
-.controller('View1Ctrl', [function() {
-
-}]);
\ No newline at end of file
+angular
+    .module('myApp')
+    .controller('productCtrl', function($scope) {
+  $scope.products = ['Danie 1', 'Danie 2', 'Danie 3'];
```
Comment (TomaszGabrysiak) [inline app/view1/view1.js]: probably sth wrong with indents here

File: app/app.js [hunk 3]
```diff
@@ -8,7 +8,8 @@ angular
       $stateProvider
           .state('view1', {
             url: '/',
```
Comment (TomaszGabrysiak) [inline app/app.js]: Maybe it's a good time to separate urls for admin part and user part of the app?

File: app/view1/view1.js (hunk 2 above)
Comment (TomaszGabrysiak) [inline app/view1/view1.js]: `products` should be a list of objects that contains `name` as well as the other 
properties. Then in repeated part if template please use `{{ product.name }}`

File: app/view1/view1.js [hunk 4]
```diff
@@ -1,14 +1,5 @@
...
+});
```
Comment (TomaszGabrysiak) [inline app/view1/view1.js]: Adding new line at the end of the file is a good practice. Don't know why :)

Comment (PatrykDawczak) [inline app/view1/view1.js]: done

File: app/view1/view1.js (hunk 2 above)
Comment (PatrykDawczak) [inline app/view1/view1.js]: done

File: app/app.js (hunk 1 above)
Comment (PatrykDawczak) [inline app/app.js]: done

File: app/view1/view1.html [hunk 5]
```diff
@@ -16,16 +16,14 @@
...
-            <button type="button" class="list-group-item">Stek</button>
-            <button type="button" class="list-group-item">Stek</button>
-            <button type="button" class="list-group-item">Stek</button>
-            <button type="button" class="list-group-item">Stek</button>
+            <button type="button" class="list-group-item" ng-repeat="product in products">{{ product.name }}</button>
         </div>
     </div>
         <div class="col-md-4">
-            <p>Lorem Ipsum jest tekstem stosowanym jako przykładowy wypełniacz w przemyśle poligraficznym. Został po raz pierwszy użyty w XV w. przez nieznanego drukarza do wypełnienia tekstem próbnej książki. Pięć wieków później zaczął być używany przemyśle elektronicznym, pozostając praktycznie niezmienionym. Spopularyzował się w latach 60. XX w. wraz z publikacją arkuszy Letrasetu, zawierających fragmenty Lorem Ipsum, a ostatnio z zawierającym różne wersje Lorem Ipsum oprogramowaniem przeznaczonym do realizacji druków na komputerach osobistych, jak Aldus PageMaker</p>
+            <p ng-repeat="product in products">{{product.description}}</p>
```
Comment (TomaszGabrysiak) [inline app/view1/view1.html]: I propose to add spaces around text in `{{}}`, it's more readable

File: app/view1/view1.js [hunk 6]
```diff
@@ -1,14 +1,16 @@
...
-}]);
\ No newline at end of file
+angular
+    .module('myApp')
+    .controller('ProductCtrl', function($scope) {
+    $scope.products = [
+      {
+        name: 'Danie 1',
+        description: 'Lorem Ipsum jest tekstem stosowanym jako przykładowy wypełniacz w przemyśle poligraficznym. Został po raz pierwszy użyty w XV w. przez nieznanego drukarza do wypełnienia tekstem próbnej książki. Pięć wieków później zaczął być używany przemyśle elektronicznym, pozostając praktycznie niezmienionym. Spopularyz ował się w latach 60. XX w. wraz z publikacją arkuszy Letrasetu, zawierających fragmenty Lorem Ipsum, a ostatnio z zawierającym różne wersje Lorem Ipsum oprogramowaniem przeznaczonym do realizacji druków na komputerach osobistych, jak Aldus PageMaker.',
+        images: [{ full: 'img/stek.jpg'}]
```
Comment (TomaszGabrysiak) [inline app/view1/view1.js]: Why such a structure? Can you explain?

**Pull request merged**